starport/services/chain/init.go

```diff
@@ -248,41 +248,48 @@ type Account struct {
 }
 
 // AddGenesisAccount add a genesis account in the chain.
-func (c *Chain) AddGenesisAccount(ctx context.Context, account Account) error {
+func (c *Chain) AddGenesisAccount(ctx context.Context, account Account, homeDir string) error {
```

Comment:
`Chain.X` methods uses `App` type to craft info about the app and determine the home directory etc. Instead of passing a homedir directly as an argument, can we deliver it through `Chain.Home()`? If this requires a refactoring we can also drop using `AddGenesisAccount` for `networkbuilder`  and execute the `add-genesis-account` command directly at the caller's side.